covid_graphs/setup.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+from setuptools import find_packages, setup
+
+setup(
+    name="covid_graphs",
+    version="1.0",
+    description="COVID-19 graph plotting",
+    install_requires=[
+        "dash",
+        "Flask",
+        "numpy",
+        "pandas",
+        "protobuf",
+        "plotly",
+        "pytest",
+        "scipy",
+
+    ],
+    dependency_links=[],
+    include_package_data=True,
+    python_requires=">=3.7",
+    packages=find_packages(),
```

Comment:
```suggestion
    packages=find_namespace_packages(),
```
And you can then drop all `__init__.py` empty files

Reply:
This is magic, had no idea it exists :)